migration/faq/index.html

```diff
@@ -316,7 +316,7 @@ <h3>Install Individual Packages</h3>
     dependencies you have compiled. For each package in that list, run:
 </p>
 
-<pre class="codehilite"><code class="language-bash" data-trim>
+<pre class="codehilite"><code class="language-bash">
```

Comment:
We are missing closing `</code>` tag.

Reply:
See: https://github.com/laminas/laminas.github.io/pull/58/commits/387e40201ee25bb0be4ee475891588422a0f02e2#diff-7e10905ce26eea322d830db37c66e7a3R273

Reply:
I am confused, this is the other place...

Reply:
I was also confused but it is fixed now.